src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -26,7 +26,7 @@ class NUnit extends ConventionTask {
     def framework
     def verbosity
     def config
-    def timeout
+    def testCaseTimeout
```

Comment:
Should mention this in the changelog, amongst the breaking changes + bump major version of the lib

Reply:
Done, updated version to 2.00